app/src/components/Tabs/Activity/direct-measure-table.tsx

```diff
@@ -0,0 +1,158 @@
+import { ActivityValue } from "@/models/ActivityValue";
+import { EditIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Icon,
+  IconButton,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverContent,
+  PopoverTrigger,
+  Table,
+  Tag,
+  TagLabel,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { TFunction } from "i18next";
+import React, { FC } from "react";
+import { FiTrash2 } from "react-icons/fi";
+import { MdModeEditOutline, MdMoreVert } from "react-icons/md";
+
+interface DirectMeasureTableProps {
+  t: TFunction;
+  activityData: ActivityValue[] | undefined;
+}
+
+const DirectMeasureTable: FC<DirectMeasureTableProps> = ({
+  activityData,
+  t,
+}) => {
+  return (
+    <Box>
+      <Table variant="simple" borderWidth="1px">
+        <Thead bg="background.backgroundLight">
+          <Tr fontSize="button.sm" fontWeight="bold">
+            <Th isTruncated>{t("building-type")}</Th>
+            <Th isTruncated>{t("data-quality")}</Th>
+            <Th isNumeric isTruncated>
+              {t("co2-emissions")}
+            </Th>
+            <Th isNumeric isTruncated>
+              {t("n2o-emissions")}
+            </Th>
+            <Th isNumeric isTruncated>
+              {t("ch4-emissions")}
+            </Th>
+            <Th></Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {activityData?.map((activity: ActivityValue, i: number) => {
+            const dataQuality = activity?.dataSource?.dataQuality;
+            return (
+              <Tr key={i}>
+                <Td isTruncated>{t(activity?.activityData?.activity_type!)}</Td>
+                <Td>
+                  <Tag p="8px" minW="50px" variant={dataQuality}>
+                    <TagLabel textTransform="capitalize">
+                      {dataQuality}
+                    </TagLabel>
+                  </Tag>
+                </Td>
+                <Td isNumeric isTruncated>
+                  {activity?.activityData?.co2_amount} tCO2
```

Comment:
Unit conversion from kg (in database) to tCO2 required here. Probably also in the activity form?